.github/workflows/pr.yml

```
@@ -0,0 +1,39 @@
+name: orka-go-ci
+
+on:
+  pull_request:
+    branches:
+      - main
```

```suggestion
on:
  pull_request:
    branches:
      - main
  push:
    branches:
      - main
```

Usually we add a trigger for push to the main branch as well ... so something like this